src/pages/Profile/index.tsx

```diff
@@ -34,7 +34,7 @@ export default function Profile({ legacy = false }) {
     }
   }
 
-  if (!data.published) return <Unpublished />;
+  // if (!data.published) return <Unpublished />;
```

Comment:
Left this just commented out, since we may need this concept for ASTs and DAFs in the future.